app/assets/javascripts/application.js

```diff
@@ -15,3 +15,4 @@
 //= require a11y-dialog/a11y-dialog.min.js
 //= require accessible-autocomplete/dist/accessible-autocomplete.min.js
 //= require_tree .
+//= stub js_check.js
```

Comment:
What does `stub` do here?

Reply:
It excludes the file from the asset bundle. We don't want it in there as we're including it inline in the layout, otherwise the JS would get executed twice – see https://github.com/rails/sprockets#stub

Reply:
I've updated the commit message with this info!